frontend/src/Dashboard/CrossingListPage/CrossingList.js

```diff
@@ -39,10 +39,20 @@ class CrossingList extends React.Component {
       return (<div>Error Loading Crossings</div>);
     }
 
+    var sortedCrossings = crossings.slice();
+    sortedCrossings.sort((c1, c2) => {
+      if (c1.statusUpdateByLatestStatusUpdateId.createdAt > c2.statusUpdateByLatestStatusUpdateId.createdAt) {
```

Comment:
This is wordy. Could see there being two `const`s, `c1CreatedAt` & `c2CreatedAt`. That would add more lines but simplify what is happening in my head. No strong opinion though.